public/app/dashboard/profileController.js

```diff
@@ -125,6 +125,22 @@
     };
 
     /**
+     * Toggle activity breakdown selection (contributions or impacts).
+     *
+     * @param  {String}         filter          Which portion of the breakdown to display
+     * @return {void}
+     */
+    var toggleBreakdown = $scope.toggleBreakdown = function(filter) {
+      $scope.breakdown.selected = filter;
+
+      // Track toggle.
+      analyticsService.track('Toggle breakdown filter', {
+        'profile_user': $scope.user.id,
```

Comment:
@paulkerschen As we know, `$scope.user` can be `me` or a different user I'm browsing. Do we see that distinction in MixPanel?